io.github.theforceengine.tfe.yml

```diff
@@ -0,0 +1,69 @@
+app-id: io.github.theforceengine.tfe
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: io.github.theforceengine.tfe.sh
+
+finish-args:
+  # hardware 3D
+  - --device=dri
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Audio
+  - --socket=pulseaudio
+  # Access to the default Steam Dark Forces data directory
+  - --filesystem=~/.local/share/Steam/steamapps/common/Dark Forces/Game/
```

Comment:
how does this work of one uses the flatpak version of steam?

Also does it work the the space in the filename?

Reply:
For a regular, non-Flatpak/Snap installation of Steam, this works fine even with the space in the path - I've now extended the permissions and launcher wrap script to also check the default library path for the Flatpak version of Steam for the game data.

There are a number of hard to cover cases for the detection here - if e.g. the user has the game installed to another Steam library that isn't in the default location (say, on the SD card on a Steam Deck) this path will also be different and the auto-detection will fail.

Users can always copy the game data to `~/.var/app/io.github.theforceengine.tfe/data/` if the auto-detection does not work (and will in fact be instructed to do so by the launcher script); I just wanted to cover the "default" option.